env.sh

```diff
@@ -1,5 +1,14 @@
 #!/bin/bash
 
+get_baseurl () {
+  local proto=$(echo ${1} | grep :// | sed -e's,^\(.*://\).*,\1,g')
```

Comment:
What's the function of the grep here? Echoing straight into the sed would work just as well afaik.

Reply:
Good point. I will get rid of the grep.